CHANGES.txt

```diff
@@ -21,8 +21,6 @@ Fixes
   (#848)
 * Fix demo application description (#850)
 * Fix auto-add functionality in ListStrModel (#860)
-* Move format_func, format_str and invalid traits from factory to editor
-  (#859, #900)
```

Comment:
i think we only reverted #859 , not #900 as well. right?

Reply:
That's right.
The issue fixed by #900 was directly caused by #859 though. With #859 reverted, #900 isn't fixing anything visible to the users. We did not need to revert #900 either, because the change there can stay in its own right (it is correcting a function signature trying to ducktype the editor class's `__init__`).